cluster/manifests/nvidia/nvidia-gpu-device-plugin.yaml

```diff
@@ -42,27 +42,23 @@ spec:
       - name: device-plugin
         hostPath:
           path: /var/lib/kubelet/device-plugins
-      - name: dev
-        hostPath:
-          path: /dev
       containers:
       - name: nvidia-gpu-device-plugin
-        image: registry.opensource.zalan.do/teapot/nvidia-gpu-device-plugin:4baa941d8df91e5dc1736adea74ce54e564dd782
-        command:
-        - /usr/bin/nvidia-gpu-device-plugin
-        - -host-path=/opt/nvidia
-        - -logtostderr
+        image: registry.opensource.zalan.do/teapot/nvidia-gpu-device-plugin:v0.10.0
+        args:
+        - --fail-on-init-error=false
+        - --pass-device-specs
         resources:
           requests:
             cpu: 50m
-            memory: 10Mi
+            memory: 25Mi
           limits:
             cpu: 50m
-            memory: 10Mi
+            memory: 25Mi
```

Comment:
The plugin got OOMKilled on some instance types. That's why I bumped it here.

Do we need to adjust the the reserved resources for DaemonSets for this?

Reply:
We might have to bump `teapot_admission_controller_daemonset_reserved_memory` by `25Mi` in all clusters to be absolutely on the safe side I think.